### PR TITLE
Component/ellipsis loader

### PR DESCRIPTION
**Pull request by musangowope**
nuked!!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "node-sass": "^4.12.0",
     "react": "~16.8.1",
     "react-dom": "~16.8.1",
-    "react-scripts": "2.0.5"
+    "react-scripts": "2.0.5",
+    "prop-types": "latest"
```
Comment (sscholle) [inline package.json]: can we rather use a version number: `latest` can introduce breaking changes

Comment (sscholle) [reply package.json]: allianz and assa
```
"prop-types": "^15.6.0"
```

File: src/components/EllipsisLoader/index.test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import EllipsisLoader from './index';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Ellipsis Loader', () => {
+  it('loader__lds-ellipsis__dot should have background color #666', () => {
```
Comment (sscholle) [inline src/components/EllipsisLoader/index.test.js]: this is only a single test, we need to add at least another one to fulfil an OKR requirement
1. we could test PropTypes or
2. we could test that it renders 4 dots

Comment (musangowope) [reply src/components/EllipsisLoader/index.test.js]: tested option 2

File: package.json (hunk 1 above)
Comment (musangowope) [reply package.json]: done

**Comment by sscholle** [comment]
@musangowope still waiting for changes...

**Pull request closed without merging**